### DISCUSSION TITLE
Download Full Installer HP Zbook 15 G2 Opencore 0.8.0

### DISCUSSION THREAD

**Discussion by HNSafes** [HNSafes/Zbook-15-G2#8] [Kho Download]
# ✏️  Chú ý: 
# 📫 Opencore Laptop HP Zbook 15 G2 - 0.8.0 Full Installer Fix All 🥰.              
 
#VNSafes chia sẻ cho anh em full bộ cài đặt macOS Monterey 12.3.1 Opencore dành cho laptop HP Zbook 15 G2, ae nào chung cấu hình với mình thì có thể tải về chơi nhé.

• Cấu hình máy: Zbook 15 G2.                             
• CPU: Core I7 4810 QM 4TH.                                      
• CARD: HD Graphics 4600.                                        
• RAM: 8GG BUS 1600......                      

Đây là bộ EFI mình tự build chuẩn nhất hệ mặt trời 😁😁🤪. 
Dell M4800 cũng có thể dùng chung, chỉ cần chỉnh sửa config.plist một chút là có thể chạy nhé.                                                  
Link Down: [Download Full Installer macOS Monterey 12.3.1 Laptop Zbook 15 G2 Opencore](https://drive.google.com/drive/u/2/folders/1D2aPrGwLULIiQi9aHAe_f1NG5pIVAHx4 )                       
![Ảnh chụp Màn hình 2022-04-19 lúc 12 34 35](https://user-images.githubusercontent.com/103987160/163943326-f0feba96-c7ee-4e5e-8a48-26573a480818.png)

**Comment by originalshop019** [comment]
Thank u for this wonderful work, could u tell me the bios modification that i should change before launching  the installation process
thank u
